src/components/widgets/AccordionList.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import DataAccordionList from '@/data/AccordionList.json';
+
+function AccordionList() {
+  // Put require data into new variable
+  const data = DataAccordionList[0].data.items;
+  // State to keep track of selected items
+  const [selected, setSelected] = useState([]);
+
+  // Function to toggle selection of an item
+  const toggle = (i) => {
+    if (selected.includes(i)) {
+      // If item is already selected, remove it from the selected array
+      setSelected(selected.filter((item) => item !== i));
+    } else {
+      // If item is not selected, add it to the selected array
+      setSelected([...selected, i]);
+    }
+  };
+
+  return (
+    <>
+      <div className="grid grid-cols-2 gap-4">
+        {data.map((item, i) => (
+          <div key={i} className="">
+            <div className="flex flex-col rounded-2xl bg-[#F7F7F7] pt-4 px-4" onClick={() => toggle(i)}>
```

Comment:
use tailwind default classes. for example `bg-gray-50` or customize config and add new set of bg.

```suggestion
            <div className="flex flex-col rounded-2xl bg-gray-50 pt-4 px-4" onClick={() => toggle(i)}>
```